packages/view/src/ide/FakeIDEAdapter.ts

```diff
@@ -14,7 +14,7 @@ export default class FakeIDEAdapter implements IDEPort {
     const onReceiveMessage = (e: IDEMessageEvent): void => {
       const responseMessage = e.data;
       const { command, payload } = responseMessage;
-      const payloadData = payload ? JSON.parse(payload) : undefined;
+      const payloadData = command && payload ? JSON.parse(payload) : undefined;
 
       switch (command) {
         case "fetchAnalyzedData":
```

Comment:
살펴보니, onReceiveMessage가 githru가 보낸 것 뿐만이 아니라 모든 종류의 이벤트들을 다 받는데,
그 과정에서 payload 데이터 형태를 인식하는 부분이 달라져서 일어나는 일입니다.

githru command일때만 payload를 읽게 하면 될 것 같습니다.

```suggestion
            const { command, payload } = responseMessage;

      switch (command) {
        case "fetchAnalyzedData":
          return events.handleChangeAnalyzedData(payload ? JSON.parse(payload) : undefined);
        case "fetchBranchList":
          return events.handleChangeBranchList(payload ? JSON.parse(payload) : undefined);
        default:
          console.log("Unknown Message");
      }
```